data/json/items/armor/cloaks.json

```
@@ -437,6 +437,1070 @@
+        "append": true
+      },
+      {
+        "id": "tongan_flag",
+        "name": { "str": "Tongan flag" },
+        "description": "A flag of a red field with the white rectangle on the upper hoist side corner bearing the red Greek Cross in the center.",
+        "weight": 10,
+        "append": true
+      },
+      {
+        "id": "trinidad_and_tobago_flag",
+        "name": { "str": "Trinidad and Tobago flag" },
+        "description": "A flag of a red field with a white fimbriated black diagonal band from the upper hoist side to the lower fly side.",
+        "weight": 10,
+        "append": true
+      },
+      {
+        "id": "tunisian_flag",
+        "name": { "str": "Tunisian flag" },
+        "description": "A flag of a red field with a centered white sun disc containing a red five pointed star surrounded by a red crescent.",
+        "weight": 10,
+        "append": true
+      },
+      {
+        "id": "turkish_flag",
+        "name": { "str": "Turkish flag" },
+        "description": "A flag of a red field with a white star and crescent slightly left of center.",
+        "weight": 10,
+        "append": true
+      },
+      {
+        "id": "turkmen_flag",
+        "name": { "str": "Turkmen flag" },
+        "description": "A flag of a green field with a vertical red stripe near the hoist side, containing five carpet guls stacked above two crossed olive branches; a white waxing crescent moon and five white five pointed stars appear in the upper field, to the fly side of the red stripe.",
+        "weight": 10,
+        "append": true
+      },
+      {
+        "id": "tuvaluan_flag",
+        "name": { "str": "Tuvaluan flag" },
+        "description": "A flag of a light blue field with a union jack in the canton and nine yellow stars on the fly half of the flag.",
+        "weight": 10,
+        "append": true
+      },
+      {
+        "id": "ugandan_flag",
+        "name": { "str": "Ugandan flag" },
+        "description": "A flag that features six equal horizontal bands of black, yellow, red, black, yellow, and red, a white disc is superimposed at the center with a grey crowned crane, facing the hoist side.",
+        "weight": 10,
+        "append": true
+      },
+      {
+        "id": "ukrainian_flag",
+        "name": { "str": "Ukrainian flag" },
+        "description": "A horizontal bicolor flag of blue and yellow.",
+        "weight": 10,
+        "append": true
+      },
+      {
+        "id": "emirati_flag",
+        "name": { "str": "Emirati flag" },
+        "description": "A horizontal tricolor flag of green, white and black with a vertical red bar at the hoist.",
+        "weight": 10,
+        "append": true
+      },
+      {
+        "id": "united_kingdom_flag",
+        "name": { "str": "United Kingdom flag" },
+        "description": "A flag with white fimbriated symmetric red cross on a blue field with a white fimbriated counterchanged saltire of red and white. Also known as the Union Jack.",
```

Need a double space here and in uzbek_flag for the localisation thingy to work
```suggestion
        "description": "A flag with white fimbriated symmetric red cross on a blue field with a white fimbriated counterchanged saltire of red and white.  Also known as the Union Jack.",
```